pkg/mls/store/models.go

```diff
@@ -36,4 +36,5 @@ type WelcomeMessage struct {
 	CreatedAt       time.Time `bun:",notnull"`
 	InstallationKey []byte    `bun:",notnull,type:bytea"`
 	Data            []byte    `bun:",notnull,type:bytea"`
+	HpkePublicKey   []byte    `bun:"hpke_public_key,notnull,type:bytea"`
```

Comment:
Nit: s/Hpke/HPKE

Reply:
It might add more confusion than it's worth. The generated protos are always going to be `Hpke`, so changing the model would give the DB the same field name with different casing.